_posts/2019-10-29-dhis2-and-docker.md

```diff
@@ -0,0 +1,214 @@
+---
+title: DHIS2 with docker
+layout: post
+categories: [blog, docker]
+author: gintare
+---
+
+# DHIS2 with docker
+Docker makes core-teams development and testing efforts easier every day! It allows us to spin up various environments with any DHIS2 version, numerous tomcat versions and different components, such as redis, nginx, in no time. Because we wanted to share docker with community, we made publishing docker images a part of our delivery pipeline. This guide aims to provide some guidelines on how to use docker to quickly set up DHIS2. 
+
+## Prerequisites
+This guide is aimed at people with at least minimal knowledge about docker. If you don't know too much about docker just yet, but think that it might be useful to you, I encourage you to take a look at resources listed in [resources](#Resources) section. 
+
+In addition to minimal docker-knowledge, you will need: 
+- docker downloaded and installed on your machine. 
+- docker-compose downloaded and installed on your machine. 
+
+## Introduction
+
+### Vocabulary
+*Image* - a template that contains the application and all the dependencies that are required to run that application. 
+*Container* - a running instance of docker image. 
+*Dockerfile* - text file, with lines and instructions on how to build the docker image. 
+
+### Docker Hub
+
+Docker Hub is an official cloud-based repository where docker users store and distribute their images. DHIS2 has its organization on [Docker Hub](https://hub.docker.com/u/dhis2/). At the time of writing this guide, we have 4  **active** repositories: base-dev, base, core-dev, core.
```

Comment:
"An official"? I think it is safe to refer to it as "the official" as it is the only one that Docker actively endorses.